pkg/server/init.go

```diff
@@ -36,9 +36,7 @@ func NewBackupRestoreComponentConfig() *BackupRestoreComponentConfig {
 		SnapstoreConfig:          snapstore.NewSnapstoreConfig(),
 		CompressionConfig:        compressor.NewCompressorConfig(),
 		RestorationConfig:        brtypes.NewRestorationConfig(),
-		OwnerCheckConfig:         brtypes.NewOwnerCheckConfig(),
 		DefragmentationSchedule:  defaultDefragmentationSchedule,
-		EtcdProcessName:          defaultEtcdProcessName,
```

Comment:
Please also remove `defaultEtcdProcessName` whereever it is defined.

Reply:
It is define here: https://github.com/gardener/etcd-backup-restore/blob/d57caede0507002c95181d755055959a23dd3595/pkg/server/types.go#L25